app/src/main/java/io/appium/uiautomator2/model/api/gestures/FlingModel.java

```diff
@@ -0,0 +1,47 @@
+package io.appium.uiautomator2.model.api.gestures;
+
+import android.graphics.Rect;
+
+import androidx.test.uiautomator.Direction;
+
+import java.util.Arrays;
+
+import io.appium.uiautomator2.model.RequiredField;
+import io.appium.uiautomator2.model.api.BaseModel;
+import io.appium.uiautomator2.model.api.ElementModel;
+
+public class FlingModel extends BaseModel {
+    public ElementModel origin;
+    public Double startX;
+    public Double startY;
+    public Double width;
+    public Double height;
+    @RequiredField
+    public String direction;
+    public Integer speed;
+
+    public FlingModel() {}
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)